psi4/driver/p4util/p4regex.py

```diff
@@ -29,8 +29,8 @@
 from __future__ import absolute_import
 import re
 
-yes = re.compile(r'^(yes|true|on|1)', re.IGNORECASE)
-no = re.compile(r'^(no|false|off|0)', re.IGNORECASE)
+yes = re.compile(r'^(yes|true|on|1\.?0*$)', re.IGNORECASE)
+no = re.compile(r'^(no|false|off|0\.?0*$)', re.IGNORECASE)
```

Comment:
I'm uncomfortable with `1.0` turning `True`. Undo these, I think, and add `if isinstance(lvalue, float):\n\tpass` [here](https://github.com/psi4/psi4/blob/master/psi4/driver/p4util/procutil.py#L77).

Reply:
Well, it was returning true before for all numerals starting with 1 and 0... Is the current version (matching only "1" and "0" - no floats) acceptable?